README.md

```diff
@@ -77,13 +77,13 @@ rm -rf /var/lib/microshift && rm -r $HOME/.microshift
 In production environment MicroShift can be deployed as:
 
 1. Install via an RPM, utilizing a host-provided cri-o runtime and be lifecycle-managed by systemd
-2. [Install as a container via Podman, utilizing cri-o runtime and be lifecycle-managed by systemd](./docs/microshift-containerized/README.md)
+2. [Install as a container via Podman, utilizing cri-o runtime and be lifecycle-managed by systemd](https://microshift.io/docs/known-issues/known-issues/)
```

Comment:
update the url for microshift-containerized

Reply:
bad paste...good catch